### PR TITLE
Ui changes

### PR DESCRIPTION
**Pull request by kush3107**
I have changed the basic layout of home screen here. The tab bar has been replaced by bottom navigation bar and the filter icon is hidden when on the stats tab.

### CODE REVIEW

File: mobx/lib/model/todo_list.dart [hunk 1]
```diff
@@ -6,6 +6,8 @@ part 'todo_list.g.dart';
 
 class TodoList = _TodoList with _$TodoList;
 
+enum AppTab { todos, stats }
```
Comment (pavanpodila) [inline mobx/lib/model/todo_list.dart:9]: How about the name `TabType` ?

**Review by pavanpodila — CHANGES_REQUESTED** [review]

File: mobx/lib/home_page.dart [hunk 2]
```diff
@@ -7,102 +7,124 @@ import 'package:mobx_sample/model/todo_list.dart';
 import 'package:mobx_sample/todo_details_page.dart';
 import 'package:provider/provider.dart';
 
-class HomePage extends StatelessWidget {
+class HomePage extends StatefulWidget {
```
Comment (pavanpodila) [inline mobx/lib/home_page.dart:10]: Let's keep this a StatelessWidget, this tab does not have to know what tab it belongs to. That should be controlled at a higher level

**Review by pavanpodila — CHANGES_REQUESTED** [review]

File: mobx/lib/model/todo_list.dart (hunk 1 above)
Comment (kush3107) [reply mobx/lib/model/todo_list.dart:9]: Yea that makes more sense. I'll make the change.

File: mobx/lib/home_page.dart (hunk 2 above)
Comment (kush3107) [reply mobx/lib/home_page.dart:10]: So you are suggesting that I should keep the activeTab in the mobx store maybe a layout store?

Comment (pavanpodila) [reply mobx/lib/home_page.dart:10]: Ya you can keep an observable in the store like `currentTab` or `activeTab` and switch whenever that changes with an `Observer`

Comment (kush3107) [reply mobx/lib/home_page.dart:10]: Okay I will make the changes

**Comment by pavanpodila** [comment]
Is this PR still the latest or the other one #2 ?

**Comment by kush3107** [comment]
@pavanpodila This is not the latest PR other one is.

**Pull request closed without merging**